core/server/web/shared/middlewares/error-handler.js

```diff
@@ -73,6 +86,20 @@ _private.JSONErrorRenderer = (err, req, res, next) => { // eslint-disable-line n
     });
 };
 
+_private.JSONErrorRendererV2 = (err, req, res, next) => { // eslint-disable-line no-unused-vars
+    res.json({
+        errors: [{
+            message: err.message,
+            context: err.context,
+            type: err.errorType,
+            details: err.errorDetails,
+            help: err.help,
+            code: err.code,
+            id: err.id
+        }]
```

Comment:
@gargol If any of the values is undefined, we should ensure we still return the full format. COuld you please check that?

Reply:
Hm. Ok, makes sense. At the moment it does't return `undefined`s. Will default to those values to return`null` instead :+1: